.pylintrc

```diff
@@ -41,7 +41,7 @@ fail-under=10
 #from-stdin=
 
 # Files or directories to be skipped. They should be base names, not paths.
-ignore=CVS
+ignore=CVS, setup.py
```

Comment:
Not sure why `CVS` is needed here. Should I remove it?

Reply:
Yes please